eogrow/tasks/testing.py

```diff
@@ -66,24 +79,78 @@ def execute(self, eopatch: EOPatch, seed: int | None = None) -> EOPatch:
         return eopatch
 
 
-class DummyTimestampFeatureTask(EOTask):
+class GenerateRasterFeatureTask(EOTask):
```

Comment:
Just to mention that we already have some similar code for the same functionality in https://github.com/sentinel-hub/eo-learn/blob/develop/core/eolearn/core/utils/testing.py#L47. Perhaps this was missed, or perhaps the functionality is different. Would it make sense to make the two parts of code more aligned?

Reply:
the functionality in eo-learn focuses on generating dummy eopatches, but here we focus on generating features. Some key differences:
- allows per-feature configuration
- allows different distributions
- more precise configuration

in eo-learn we have no need of that, so I wouldn't join them too much